apps/src/applab/manageAssets/assetRow.jsx

```diff
@@ -0,0 +1,120 @@
+var React = require('react');
+var AssetsApi = require('./clientApi');
+
+var defaultIcons = {
+  image: 'fa fa-picture-o',
+  audio: 'fa fa-music',
+  video: 'fa fa-video-camera'
+};
+
+function getThumbnail(type, name) {
+  switch (type) {
+    case 'image':
+      var src = '/v3/assets/' + dashboard.project.current.id + '/' + name;
+      return <img src={src} style={{
+        width: 'auto', maxWidth: '100%', height: 'auto', maxHeight: '100%',
+        zoom: 2, marginTop: '50%', transform: 'translateY(-50%)'
+      }}/>;
+    default:
+      return <i className={defaultIcons[type] || 'fa fa-question'}
```

Comment:
Should fa fa-question also be in defaultIcons, and then we do defaultIcons[type] || defaultIcons.question?